database/helpers.js

```diff
@@ -0,0 +1,48 @@
+const RecommendedItem = require('./RecommendedItem.js');
+
+const formatName = (string) => {
+  return string[0].toUpperCase() + string.split('').slice(1).join('');
+};
+
+const getDept = (dept, callback = (result) => result) => {
+  let department = formatName(dept);
+  return RecommendedItem.find({ department }, callback);
```

Comment:
Is a return necessary here? Not that it is causing any problems. Just curious.

Reply:
No, not at all. Good catch that was a relic of multiple trial and errors! I'll remove them.